docker-compose.test.yml

```diff
@@ -0,0 +1,14 @@
+version: '2.1'
+
+services:
+  db:
+    image: postgres
+    env_file: env.d/test
+  web:
+    build:
+      context: .
+      args:
+        DEVELOPMENT: "true"
```

Comment:
Well actually, this could be `IS_NOT_PRODUCTION` instead of `DEVELOPMENT` or `IS_DEV`.